### PR TITLE
Skeleton fixing

### LINKED ISSUE #20: Fix package.json

### PR DESCRIPTION
**Pull request by Smyku6**
closes #20

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,68 +2,81 @@
...
+    "build": "parcel build",
+    "format": "",
+    "start:dev": "",
+    "test": "",
+    "test:watch": "",
+    "test:cov": ""
   },
   "repository": {
     "type": "git",
     "url": "git+https://github.com/CodersCamp2020/CodersCamp2020.Project.JavaScript.StarWarsQuiz.git"
```
Comment (vieraboschkova) [inline package.json:22]: Maybe we can change that line also?

File: package.json [hunk 2]
```diff
@@ -2,68 +2,81 @@
...
   },
   "keywords": [],
-  "author": "CodersCamp2020",
+  "author": {
+    "name": "CodersCamp2021",
+    "email": "kontakt@coderscrew.pl",
```
Comment (vieraboschkova) [inline package.json:27]: Perhaps an email of one of the team members?

File: package.json [hunk 3]
```diff
@@ -2,68 +2,81 @@
...
+    "url": "https://github.com/CodersCamp2021"
+  },
+  "contributors": [
```
Comment (vieraboschkova) [inline package.json:30]: I like it a lot that you took the time to add everyone! 💯

File: package.json [hunk 4]
```diff
@@ -2,68 +2,81 @@
...
+    {
+      "name": "Adam Smycz",
+      "url": "https://github.com/Smyku6"
+    }
+  ],
   "license": "ISC",
   "bugs": {
     "url": "https://github.com/CodersCamp2020/CodersCamp2020.Project.JavaScript.StarWarsQuiz/issues"
   },
   "homepage": "https://github.com/CodersCamp2020/CodersCamp2020.Project.JavaScript.StarWarsQuiz#readme",
```
Comment (vieraboschkova) [inline package.json:60]: Consider changing :)

**Review by vieraboschkova — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (Smyku6) [reply package.json:22]: I will remove it for now

File: package.json (hunk 2 above)
Comment (Smyku6) [reply package.json:27]: I will add an email to our Development Manager

File: package.json (hunk 4 above)
Comment (Smyku6) [reply package.json:60]: true!

File: package.json (hunk 3 above)
Comment (Smyku6) [reply package.json:30]: I forgot about the mentor 🙇

**Review by vieraboschkova — APPROVED** [review]
Good job! It is ready for further work now!

**Pull request merged**